### PR TITLE
fix(bucket): return documents whose relation fields are empty

### PR DESCRIPTION
**Pull request by thesayyn**

### CODE REVIEW

**Comment by thesayyn** [comment]
@htuna07 First commit is irrelevant to the changes that supposed to have been done in this bugfix.

**Comment by thesayyn** [comment]
Tests are failing. I'll look into them tomorrow.

File: .bazelrc [hunk 1]
```diff
@@ -47,3 +47,7 @@ build --host_force_python=PY2
 run --host_force_python=PY2
 
 common --experimental_allow_incremental_repository_updates
+
+
+# Disable Test Caching
+test --nocache_test_results
```
Comment (htuna07) [inline .bazelrc]: Why do we need to disable test caching?

File: packages/database/testing/testing.module.ts [hunk 2]
```diff
@@ -48,98 +49,75 @@ export class DatabaseTestingModule implements OnModuleDestroy {
...
-      ...config.members[2],
-      hidden: true,
-      priority: 0,
-      slaveDelay: 5,
-      tags: {slaveDelay: "true"}
-    };
-    await dbService.executeDbAdminCommand({replSetReconfig: config});
-    //wait until replicaSet reconfigured
-    await new Promise(resolve => setTimeout(resolve, 10000));
-  }
```
Comment (htuna07) [inline packages/database/testing/testing.module.ts]: History service needs to this delay settings if I'm not wrong.

File: .bazelrc (hunk 1 above)
Comment (thesayyn) [reply .bazelrc]: Some of the test suits are affecting each other due to resource usages or something else. When a test doesn't run along with the affected test, it passes. That's why we want to run tests every time to ensure tests are passing together.

File: packages/database/testing/testing.module.ts (hunk 2 above)
Comment (thesayyn) [reply packages/database/testing/testing.module.ts]: It didn't fail when I ran the tests so I decided to delete these lines.

**Comment by thesayyn** [comment]
All checks have passed. God may be with us 🥇

**Pull request merged**